10-JavaScript/ajax-post.md

```diff
@@ -1,99 +1,185 @@
 # JavaScript: AJAX Post
 
 ## Learning Goals
-- Use jQuery to make AJAX POST requests
+- Use axios to make AJAX POST requests
+- Use JavaScript to read form data
+- Handle API errors gracefully in a JavaScript SPA
 
-Just like we used AJAX to send GET requests, we can also use it to send POST requests. Based on our knowledge of POST requests, what would you say is the main thing we would add to a POST request that we did not include in a GET request? YES! Data!
+## Introduction
+
+Just like we used axios and AJAX to send GET requests, we can also use it to send POST requests.
+
+**Question:** Thinking back to our previous work with APIs in Ruby, what are the big differences between POST and GET?
+
+## Making a POST request
+
+Let's get right to it! We will make a POST request to add a pet to the list. Read through the [API documentation on GitHub](https://github.com/AdaGold/pets_api/blob/master/README.md), and answer the following questions:
+
+- What endpoint should we use?
+- What information can we send?
+- What information **must** we send?
+- How do we need to structure the data we send?
+
+
+This lesson will continue to use the [HTML in the reference section](reference/axios/index.html). Notice the `<form>`. When the user submits that form, we'll want to read all the data and send a POST request to the API.
+
+Let's start with some jQuery to make that connection.
 
-### POST request
-POST requests are set up very similar to GET requests.
 ```javascript
-const url = 'https://petdibs.herokuapp.com/pets';
-const data = {
-  name: 'Chestnut',
-  age: 17,
-};
-const callback = function callback() {
-  console.log('success!');
+// Inside $(document).ready() in index.js
+$('#pet-form').submit(createPet);
+```
+
+Next, we need to build the `createPet` method. We'll start out with hard-coded data, and later on we'll expand our code to read values from a form.
+
+```javascript
+// Above $(document).ready() in index.js
+const createPet = () {
+  // Later we'll read these values from a form
+  const petData = {
+    name: 'Socks',
+    age: 7,
+    owner: 'Bill'
+  };
+
+  reportStatus('Sending pet data...');
+
```

Comment:
You don't seem to be disabling the form submit so it refreshes the page.  i.e. `event.preventDefault()`.

That's not a bad thing, as showing them **why** that doesn't work is a good idea.

Reply:
Whoops, definitely a typo!